libs/filabridge/src/SibGenerator.cpp

```diff
@@ -66,12 +66,26 @@ SamplerInterfaceBlock const& SibGenerator::getPerViewSib(uint8_t variantKey) noe
     return v.hasVsm() ? sibVsm : sibPcf;
 }
 
+SamplerInterfaceBlock const& SibGenerator::getPerRenderPrimitiveMorphingSib(uint8_t variantKey) noexcept {
+    using Type = SamplerInterfaceBlock::Type;
+    using Format = SamplerInterfaceBlock::Format;
+
+    static SamplerInterfaceBlock sib = SamplerInterfaceBlock::Builder()
+            .name("Morphing")
+            .add("targets", Type::SAMPLER_2D_ARRAY, Format::FLOAT)
```

Comment:
Should precision be set to `HIGHP`?

Reply:
`MEDIUMP` was enough when I tested on macOS, iOS and Android. If you want to set `HIGHP`precision I will do that.

Reply:
Generally we use `highp` for position-related data. @pixelflinger what do you think?

Reply:
It definitely should be `HIGHP` -- if MEDIUMP was really enough, the texture could be RGBA16F. My guess is that the driver doesn't honor MEDIUMP properly.

Reply:
Ok It's done. As far as I know, the driver doesn't have mediump. They have only lowp and highp.